cms/search/templates/search/search.html

```diff
@@ -8,7 +8,12 @@
 {% block content %}
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-full">
-      <h1 class="nhsuk-heading-l">Search results for: Condition</h1>
+      <h1 class="nhsuk-heading-l">
+        Search results for:
+        {% if query_string %}
```

Comment:
I think this `if` statement is in the wrong place - if there isn't a query we'll end up with "Search results for:" and nothing after it - maybe wrap this around the entire search string block?

Reply:
> "Search results for: kittens"
Sorry: what if "kittens" is instead just the empty string, is "Search results for: " then nothing acceptable?

> thats correct
dont show

Reply:
https://dxw.slack.com/archives/C027VJH9S3A/p1649329060036969?thread_ts=1649328175.241669&cid=C027VJH9S3A

Fix on the way